install.sh

```diff
@@ -1,75 +1,80 @@
-#!/bin/sh
+#!/usr/bin/env sh
 # Copyright 2022 khanhas. GPL license.
 # Edited from project Denoland install script (https://github.com/denoland/deno_install)
 
 set -e
 
 case $(uname -sm) in
-"Darwin x86_64") target="darwin-amd64" ;;
-"Darwin arm64") target="darwin-arm64" ;;
-"Linux x86_64") target="linux-amd64" ;;
-*)
-	echo "Unsupported platform $(uname -sm). Only Darwin x86_64, Darwin arm64 and Linux x86_64 binaries are available."
-	exit
-	;;
+	"Darwin x86_64") target="darwin-amd64" ;;
+	"Darwin arm64") target="darwin-arm64" ;;
+	"Linux x86_64") target="linux-amd64" ;;
+	*) echo "Unsupported platform $(uname -sm). Only Darwin x86_64, Darwin arm64 and Linux x86_64 binaries are available."; exit ;;
```

Comment:
We actually already support linux-arm64 in the upgrade command, so I'm not sure why we are ignoring it 🤷

Reply:
because i hadn't seen it. 
also, i'm not sure what the exact `arm64` in uname is called but ill just make it `Linux arm64`.